resources/views/components/tables/desktop/exchanges.blade.php

```diff
@@ -7,7 +7,7 @@
             <x-tables.headers.desktop.text name="general.exchange.top_pairs" />
 
             <x-tables.headers.desktop.number name="general.exchange.price">
-                <span>({{ Settings::currency()}})</span>
+                <span>(@lang('currencies.usd.currency'))</span>
```

Comment:
My bad, it's a config file :hide:

```suggestion
                <span>({{ config('currencies.usd.currency') }})</span>
```